src/layers/core/icon-layer/icon-layer.js

```diff
@@ -60,7 +60,8 @@ const defaultProps = {
   getPosition: x => x.position,
   getIcon: x => x.icon,
   getColor: x => x.color || DEFAULT_COLOR,
-  getSize: x => x.size || 1
+  getSize: x => x.size || 1,
+  getAngle: x => x.angle || 0
```

Comment:
> Let's make it degrees **for now** 

As long as we don't publish this in a release (including 4.0 patch), that's OK.

If this is going into official release it needs to be audited like all other props.